cs/v1/CS_v1.0.0.yaml

```diff
@@ -112,7 +112,7 @@ paths:
       parameters:
         - schema:
             type: string
-            pattern: ^[A-Z]{2}[A-Z2-9]{3}$
+            pattern: '^[A-Z]{2}[A-Z2-9]{3}$'
```

Comment:
You are adding single-quotes to all patterns? This is in theory ok here since there are no characters that need escaping - but there are some places where the expression is now wrong. To me it seems like an "automatic" modification done by StopLight :-(

Reply:
Actually I have removed quotes from all the patterns to avoid confusion but it has been added by stoplight when I did a commit. Will remove from all places.